### ISSUE TITLE
Allow tests to provide values for AutoFields

### ISSUE THREAD

**Issue by groovecoder** [vandersonmota/model_mommy#213]
I want to force an `id` value in a mommy model so I can assert against it later in the test. E.g.:
```
      @fudge.patch('auctions.models.send_mail')
      def test_mail_contains_claim_by_bid_url(self, mock_send_mail):
          user = mommy.make(settings.AUTH_USER_MODEL)

          offer_bid = mommy.prepare(
              'auctions.Bid', id=9, offer=100, user=user, ask=1000, url=self.url
          )

          mock_send_mail.expects_call().with_args(
              arg.any(),
              arg.contains(reverse(
                  'custom-urls:claim-by-bid',
                  kwargs={'bid': offer_bid.id}
              )),
              arg.any(),
              ['user2@test.com']
          )
```
But it looks like [`_make` skips `AutoField`s like `id`](https://github.com/vandersonmota/model_mommy/blob/2ba228e2cb8149d6cf8554938be8655c719c1fa8/model_mommy/mommy.py#L290-L291)?

**Comment by vandersonmota** [comment]
Indeed. https://github.com/vandersonmota/model_mommy/blob/master/model_mommy/mommy.py#L290

For now, you can use as workaround prepare and set de id in the next line, then saving :)

**Comment by romulocollopy** [comment]
I coundn't reproduce the issue in the tests.
This test is passing:
```
class FillAutoFieldsTestCase(TestCase):

    def test_fill_autofields_with_provided_value(self):
        dummy = mommy.make(DummyEmptyModel, id=237)
        self.assertEqual(dummy.id, 237)

    def test_keeps_prepare_autovalues(self):
        dummy = mommy.prepare(DummyEmptyModel, id=543)
        self.assertEqual(dummy.id, 543)
        dummy.save()
        self.assertEqual(dummy.id, 543)
```
Is this issue yet open?

**Comment by berinhard** [comment]
Maybe we should just add a test that verifies this functionality and merge it so we could ensure its behaviour.

**Comment by romulocollopy** [comment]
Please check PR #230

**Comment by berinhard** [comment]
Fixed by #230.
